tst/EnergyPlus/unit/InputProcessor.unit.cc

```diff
@@ -1485,6 +1485,23 @@ namespace EnergyPlus {
 			EXPECT_TRUE( RepObjects( 5 ).TransitionDefer );
 		}
 
+		TEST_F(InputProcessorFixture, MisleadingIDDWarningTest)
+		{
+			std::string const idd_objects = delimited_string({
+				"Version,8.4;",
+			});
+
+			bool errors_found = false;
+			ASSERT_FALSE(process_idd(idd_objects, errors_found));
+			ProcessInput();
+
+			std::string const expected_string = delimited_string({
```

Comment:
@mbadams5 If you have any thoughts on this, your comments would be appreciated.

@rglazer2 So I think your unit test should be something like this:

```c++
// create the incorrect idd contents string
idd_contents = delimited_string({
"!IDD_Version 8.1.0",
"!IDD_BUILD 111111111"});

// process the idd objects
ASSERT_FALSE(process_idd(idd_objects, errors_found));

// create the idf contents that will fail
idf_contents = delimited_string({"Some,bad,idf,object;"});

// process the idd objects
ASSERT_FALSE(process_idf(idf_objects, errors_found));

// then check the error file
EXPECT_TRUE(compare_err_stream("something", true))
```

The idd read will succeed.  The idf read will be erroneous.  The error file should show the error of mismatched idd version.  You may also want to add the opposite test to ensure that the error isn't thrown if the IDD version **does** match when the idf is malformed.

@mbadams5 Yeah, what would go in the `compare_err_stream` call?  The entire error file contents or a specific substring?